### DISCUSSION TITLE
documentation 4 version hugo

### DISCUSSION THREAD

**Discussion by Hkr32** [twbs/bootstrap#32190] [General] [answered]
Are you planning to transform the 4 version documentation to hugo?

**Comment by XhmikosR** [comment, marked as answer]
See #32077.

Just out of curiosity, though, why do you ask? Not sure why our end users should care about it :)

**Comment by Hkr32** [comment]
Yes thank you. It was interesting. I also thought to make a translation of version 4)

**Comment by Hkr32** [comment]
@XhmikosR, great job! Is the transfer planned to the main branch or will it remain in the dev?

**Reply by XhmikosR** [reply]
The main branch is already using Hugo since almost 2 years now.

**Reply by Hkr32** [reply]
I mean version 4.5 in one docs/ folder with version 5

**Reply by XhmikosR** [reply]
I don't get what you mean, sorry. v4-dev is a separate branch which is for our 4.x version.

**Reply by Hkr32** [reply]
![image](https://user-images.githubusercontent.com/12296392/99683241-2d166900-2a91-11eb-89a3-a28f41fa3cc7.png)

**Comment by XhmikosR** [comment]
Like I said, v4-dev is a separate branch.
